table/server/underdb/glue/src/main/java/alluxio/table/under/glue/Property.java

```diff
@@ -0,0 +1,204 @@
+     * without registering it with default property list.
+     *
+     * @return udb property
+     */
+    public Property buildUnregistered() {
+      Property property = new Property(mName, mDescription, mDefaultValue);
+      return property;
+    }
+  }
+
+  /**
+   * Registers the given UDB alluxio.table.under.glue.Property to the global map.
+   *
+   * @param Property the udb property
+   * @return whether the udb property is successfully registered
+   */
+  @VisibleForTesting
+  public static boolean register(Property Property) {
+    String name = Property.getName();
+    if (DEFAULT_KEYS_MAP.containsKey(name)) {
+      return false;
+    }
+
+    DEFAULT_KEYS_MAP.put(name, Property);
+    return true;
+  }
+
+  /**
+   * Unregisters the given key from the global map.
+   *
+   * @param Property the property to unregister
+   */
+  @VisibleForTesting
+  public static void unregister(Property Property) {
+    String name = Property.getName();
+    DEFAULT_KEYS_MAP.remove(name);
+  }
+
+  public static final Property MAX_GLUE_CONNECTION =
+      new Builder(Name.MAX_GLUE_CONNECTION)
+          .setDefaultValue("5")
+          .setDescription("The maximum number of connection to glue metastore.")
+          .build();
+
+  public static final Property GLUE_REGION =
+      new Builder(Name.GLUE_REGION)
+        .setDefaultValue("")
+        .setDescription("The regional endpoint for client service calls.")
+        .build();
+
+  public static final Property CATALOG_ID =
+      new Builder(Name.CATALOG_ID)
+          .setDefaultValue("")
+          .setDescription("The catalog id of aws glue.")
+          .build();
+
+  public static final Property AWS_GLUE_ACCESS_KEY =
+      new Builder(Name.AWS_GLUE_ACCESS_KEY)
+        .setDefaultValue("")
+        .setDescription("The access key to access the aws glue.")
+        .build();
+
+  public static final Property AWS_GLUE_SECRET_KEY =
+      new Builder(Name.AWS_GLUE_SECRET_KEY)
+        .setDefaultValue("")
+        .setDescription("The secret key to access the aws glue.")
+        .build();
+
+  /**
+   * @return the name of alluxio.table.under.glue.Property
+   */
+  public String getName() {
+    return mName;
+  }
+
+  /**
+   * @return the description of a property
+   */
+  public String getDescription() {
+    return mDescription;
+  }
+
+  /**
+   * @return the default value of udb property or null if value not set
+   */
+  @Nullable
+  @Override
+  public String getDefaultValue() {
+    Object defaultValue = mDefaultValue;
+    return defaultValue == null ? null : defaultValue.toString();
+  }
+
+  /**
+   * Corresponding configurations of GLUE configurations.
+   */
+  public static final class Name {
+    // AWS Glue related properties
+    public static final String MAX_GLUE_CONNECTION = "max.connection";
+    public static final String GLUE_REGION = "aws.region";
+    public static final String CATALOG_ID = "aws.catalog.id";
```

Comment:
Did you figure out if this is required or not?

Reply:
Added to TODO.

Reply:
I don't see the TODO comment?